_faq.Rmd

```diff
@@ -1,6 +1,6 @@
 ---
-title: "PHOBILE frequently asked questions"
-author: Steven Moran and Dan McCloy
+title: "PHOIBLE frequently asked questions"
+author: Steven Moran & Daniel McCloy
```

Comment:
@drammock -- sorry about that. I should know better. I get "Steve" all the time.